skycultures/arabic/constellation_names.eng.fab

```diff
@@ -0,0 +1,138 @@
+1100 "ذو السلاح" _("The Armed One") 1,2
+1101 "الرمح" _("The Spear") 4
+1200 "السماك الأعزل" _("The Unarmed One") 1,2,3,4
+1300 "الضباع وأولاد الضباع" _("The Hyenas And Their Youngsters") 1,2,3,4
+#3 page 107
+1400 "قصعة المساكين" _("The Bowl Of The Poor") 1,2,3,4
+1500 "الروضة والأغنام" _("The Desert Garden And The Goats") 1,2,3,4
+1501 "النسق الشآمي" _("The Northern Line") 1,2,3,4
+1502 "النسق اليماني" _("The Southern Line") 1,2,3,4
+1600 "النسر الواقع" _("The Landing Eagle") 1,2,3,4
+1700 "النسر الطائر" _("The Flying Eagle") 1,2,3,4
+1800 "الفوارس" _("The Knights") 1,2,3,4
+1900 "الناقة" _("The She-Camel") 1,2,4
+1901 "رأس الناقة" _("Head Of The She-Came") 1,2,4
+1902 "عنق الناقة" _("Neck Of The She-Came") 1,2,4
+1903 "يدا الناقة" _("Legs Of The She-Came") 4
+#The name is left blank intentionally, label of 2604 Al-Thurayya is sufficient. This constellation is to show the complete Al-Thurayya image with its two hands.
+#2000 "الثريا " _("Al-Thurayya")
+2000 " " _("Al-Thurayya")
+2001 "سنام الناقة - الكف الخضيب" _("The She-Camel's Hump - The Henna Tinted Hand") 1,2,3,4
+2002 "العاتق" _("The Shoulder Blade") 1,2,3,4
+2003 "المنكب" _("The Shoulder") 1,2,3,4
+2004 "العضد" _("The Arm") 1,2,3,4
+2005 "المرفق" _("The Elbow") 1,2,3,4
+2006 "الساعد" _("The Forearm") 2,3,4
+2007 "المعصم" _("The Wrist") 1,2,3,4
+2008 "الكف الجذماء" _("The Leprous Hand") 1,2,3,4
+#3 page 83
+# TRANSLATORS: Al-Ayyuq is a proper name meaning the obstructor
+2100 "العيوق والأعلام" _("Al-Ayyuq And The Posts") 1,2,3,4
+2200 "الخباء الشآمي" _("The Northern Tent") 1,2,3,4
+2300 "القعود" _("The Young Camel") 1,2,3,4
+2400 "عناق الأرض" _("The Caracal") 1,2,3,4
+2500 "الدلو" _("The Well Bucket") 1,2,3,4
+2501 "الكرب" _("The Bucket Rope Knot") 1,2,4
+2502 "الفرغ المقدم" _("The Front Bucket Mouth") 1,2,3,4
+2504 "الفرغ المؤخر" _("The Rear Bucket Mouth") 1,2,3,4
+2506 "الرشاء" _("The Bucket Rope") 1,2,3,4
+#For reference 2 see vol. 1 pg. 166
+2550 "الحوت" _("The Fish") 1,2,3,4
+2560 "بلدة الثعلب" _("The Empty Place Of The Fox") 3,4
+2600 "الحمل" _("The Lamb") 1,2,3,4
+#This name has been corrupted in many nanuscripts and the printed references from the correct "الأيبسان" to "الأنيسان". I discussed the name in my book Reference 4.
+2601 "الأيبسان" _("The Two Forelegs Of The Lamb") 1,2,3,4
+2602 "الشرطان" _("The Two Signs") 1,2,3,4
+2603 "البطين" _("The Little Abdomen") 1,2,3,4
+# TRANSLATORS: Al-Thurayya is a proper name
+2604 "الثريا" _("Al-Thurayya") 1,2,3,4
+#This constellation defines the boundary of Al-Dabaran, his camels and two dogs. Name is intentionally left blank
+#2700 "الدبران وقلاصه وكلباه" _("Al-Dabaran, His Camels And Two Dogs")
+#2700 " " _(" ")
+2701 "الدبران" _("Al-Dabaran") 1,2,3,4
+#Ref 3 pg. 28
+# TRANSLATORS: Al-Dabaran is a proper name meaning the follower
+2702 "كلبا الدبران" _("The Two Dogs Of Al-Dabaran") 1,2,3,4
+2703 "القلاص" _("The Young She-Camels") 1,2,4
+2800 "الأسد" _("The Lion") 1,2,3,4
+2801 "الذراع المقبوضة" _("Retracted Arm") 1,2,3,4

```

Comment:
also here, add article to be consistent?

Reply:
fixed